draft-ietf-taps-interface.md

```diff
@@ -355,7 +355,7 @@ SecurityParameters := NewSecurityParameters()
 SecurityParameters.Set(identity, myIdentity)
 SecurityParameters.Set(key-pair, myPrivateKey, myPublicKey)
 
-// Specifying a Remote Endpoint is optional when using Listen()
+// Specifying a Remote Endpoint is optional when using Listen
```

Comment:
```suggestion
// Specifying a Remote Endpoint is optional when using `Listen`
```

Reply:
This is intentional this is all in code, and we can't do code voice within code voice. This is a comment in a code block, leaving as-is.